packages/schematics/angular/web-worker/schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "SchematicsAngularWebWorker",
+  "title": "Angular Web Worker Options Schema",
+  "type": "object",
+  "description": "Pass this schematic to the \"run\" command to create a Web Worker",
+  "properties": {
+    "path": {
+      "type": "string",
+      "format": "path",
+      "description": "The path at which to create the worker file, relative to the current workspace.",
+      "visible": false
+    },    
+    "project": {
+      "type": "string",
+      "description": "The name of the project.",
+      "$default": {
+        "$source": "projectName"
+      }
+    },
+    "target": {
```

Comment:
Is this needed? I don't see other supported targets

Reply:
No, it is not now. At some point it was but not anymore. Removed.